builtin/client/register.lua

```diff
@@ -1,5 +1,8 @@
-
-core.callback_origins = {}
+core.callback_origins = setmetatable({}, {
+	__index = function()
+		return {mod = "", name = ""}
```

Comment:
Not really necessary as there seems to be no `if origin then ... end` in the client code, but a decent default for it to be future-proof.